src/MAUI/Maui.Samples/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage
+    x:Class="ArcGIS.Samples.AddVectorTiledLayerFromCustomStyle.AddVectorTiledLayerFromCustomStyle"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:esri="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
+
+    <Grid>
+        <esri:MapView x:Name="MyMapView" />
+        <Button
+            Margin="5,5,5,5"
+            Clicked="StyleChooserClick"
+            HorizontalOptions="End"
+            Text="Change style"
```

Comment:
Nitpick: I think it would be beneficial to indicate to the user which tiled layer they are currently viewing. This could be displayed as a label next to the button or on the button itself.

Reply:
I agree. This is what we did on "Browse Building Floors" for reference:

![labelAndButton](https://user-images.githubusercontent.com/99750197/210846278-4c06d88f-9d12-4d1a-9afe-f856b42d3948.JPG)

Reply:
Agreed - we're already indicating to the user which layer they're viewing in WinUI and WPF implementations so we should also in MAUI.